tests/data/charm.yaml

```diff
@@ -0,0 +1,19 @@
+description: Overlay for testing current charm
+bundle: kubernetes
+applications:
+  tls-certificates:
+    charm: tls-certificates-operator
```

Comment:
As per the [README](https://github.com/canonical/tls-certificates-operator#with-self-signed-certificates-deprecated) from the `tls-certificates-operator` the self signed option is deprecated and we must switch to https://charmhub.io/self-signed-certificates

Reply:
interesting -- that charm is only in beta

Reply:
🤔 yeah, that's odd.